### ISSUE TITLE
Accessibility - Poor Color Contrast with "Sweat & Toil" Text in Banner

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#125]
The color contrast ratio of the black text ( #000000 ) and the blue banner background ( #304F6A ) is 2.5:1 and needs to be 4.5:1 or higher to pass.

If you put the text back to white ( #ffffff ), the problem will be fixed. Now, if the styling/CSS is being used by other items like the information icon at the bottom of the screen, they will need to be separate items, so they both pass a color contrast check.

![ialb - sweat and toil - home page - color contrast issues](https://user-images.githubusercontent.com/10423487/88321303-eedc8900-ccec-11ea-8963-4559732ddf2b.png)

**Comment by jfc3-dol** [comment]
Opened the application a second time, the "Sweat & Toil" text is now white, so I'm going to close this issue.
